include/LightGBM/config.h

```diff
@@ -268,6 +268,12 @@ struct Config {
   // desc = random seed for bagging
   int bagging_seed = 3;
 
+  // alias = sub_feature_bynode, colsample_bytree_bynode
+  // desc = set to ``true`` will randomly select part of features for each node
+  // desc = set to ``false`` will randomly select part of features for each true (use the same sub features for each tree)
+  // desc = **Note**: set to ``false`` can speed up the training linearly
```

Comment:
I think it's better to invert the meaning of this note, because the default value is `false`. Inattentive user may expect the speedup by setting this param to `false`, but nothing will happen because it has been already `false`.